### PR TITLE
readme: Remove line saying it isn't recommended project

### PR DESCRIPTION
**Pull request by vivekanand1101**
Signed-off-by: Vivek Anand <vivekanand1101@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,4 +32,4 @@ We will also add a bunch of more tests.
 To test the code, you will need to run the SecureDrop `make dev` command in the same system. The test suite for
 this project will test against that development container.
 
-### License: GPLv3+
\ No newline at end of file
+### License: GPLv3+
```
Comment (kushaldas) [inline README.md]: What is the actual change in this line?

Comment (vivekanand1101) [reply README.md]: \n at the end of the line was missing

**Comment by kushaldas** [comment]
@vivekanand1101 You will have to sign the commit using GNUPG.

**Comment by vivekanand1101** [comment]
will do when i am back home

**Pull request merged**